src/compute-client/src/types/dataflows.rs

```diff
@@ -592,6 +633,18 @@ impl RustType<ProtoIndexDesc> for IndexDesc {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct IndexImport {
+    /// description of index
```

Comment:
```suggestion
    /// Description of index.
```
Nit: Capitalization, doc could be more descriptive for all fields.